lib/acts_as_taggable_on/taggable/core.rb

```diff
@@ -214,7 +214,7 @@ def tagging_contexts
       self.class.tag_types.map(&:to_s) + custom_contexts
     end
 
-    def tenant
+    def taggable_tenant
       if self.class.tenant_column
         read_attribute(self.class.tenant_column)
```

Comment:
@lunaru small question: why did we specificly use `#read_attribute`?

If I have a taggable resource that is:
```rb
class MyTaggableModel
  has_one: :tenant, through: :something
  
  acts_as_taggable_on :tags
  acts_as_taggable_tenant :tenant_id
  
  def tenant_id
    something.tenant_id
  end
end
```

It just doesn't works, and that's kinda unfortunate.